From: SeongJae Park <sj@kernel.org>
Date: Sun, 1 Dec 2024 10:08:44 -0800
Subject: [PATCH] damon/core: fixup damos_walk_control return types

Both prep_fn and walk_fn return values are not really checked.  Just set
those as void for now.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 include/linux/damon.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/linux/damon.h b/include/linux/damon.h
index 29c339856ddc..b1f0fff5603c 100644
--- a/include/linux/damon.h
+++ b/include/linux/damon.h
@@ -379,8 +379,8 @@ struct damos;
  * @data:	Data that will be passed to walk functions.
  */
 struct damos_walk_control {
-	int (*prep_fn)(void *data, struct damon_ctx *ctx);
-	int (*walk_fn)(void *data, struct damon_ctx *ctx,
+	void (*prep_fn)(void *data, struct damon_ctx *ctx);
+	void (*walk_fn)(void *data, struct damon_ctx *ctx,
 			struct damon_target *t, struct damon_region *r,
 			struct damos *s);
 	void *data;
-- 
2.39.5

